Gemfile

```diff
@@ -13,7 +10,7 @@ gem 'test-unit', '~> 3.0'
 gem 'rollbar', '~> 2.7'
 gem 'skylight'
 
-gem 'sidekiq', '< 5'
+gem 'sidekiq', "~> 4"
```

Comment:
Was there a reason for this? I think `~> 4` is the same as accepting any version